src/app.js

```diff
@@ -2,15 +2,14 @@ const express = require('express');
 const cors = require('cors');
 const helmet = require('helmet');
 const createError = require('http-errors');
-const expressPino = require('express-pino-logger');
-
+const pinoHttp = require('pino-http');
 const logger = require('./logger');
```

Comment:
Add a blank line between external module imports and relative modules in the tree.